src/validateFilenames.ts

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+
+/* tslint:disable */
+require('babel-polyfill'); // tslint:disable-line
+
+import * as program from 'commander';
+import * as shelljs from 'shelljs';
+import { getFilenames } from './getFilenames';
+import { getFilenameValidationData } from './getFilenameValidationData';
+import {
+  allFilenamesAreValid,
+  prettyPrintFilenamesValidationData,
+  underline,
+} from './utils';
+
+// Setup arguments through the commander.js CLI framework
+program
+  .usage('-c [path]')
+  .option('-c, --config [path]', 'path to config file (a JS module)')
+  .parse(process.argv);
+
+// If user doesn't provide any arguments, we'll display help
```

Comment:
Can't we use fallback rules instead? Some of our repos only need camelCase and they don't have a `common.config.js` file.

Reply:
Okay, will add default rules.